webpack/webpack.config.prod.js

```diff
@@ -20,6 +20,7 @@ const clientInclude = [
 
 
 const vendor = [
+  'auth0-js',
```

Comment:
down below we can also remove the auth0-lock specific stuff

Reply:
I don't understand what you mean by this comment. Could you clarify?